### PR TITLE
Added metrics gauges for CPU and Memory limits

### PR DESCRIPTION
**Pull request by ddreier**
For #10

Here's what I came up with. I'm happy to incorporate any feedback!

Also, some other small formatting changes happened from running `go fmt`.

### CODE REVIEW

File: metrics/metrics.go [hunk 1]
```diff
@@ -6,15 +6,17 @@ import (
...
 type Metrics struct {
-	Namespace  string
-	Labels     map[string]string
-	MHzPerCore float64
-	Cores      int
+	Namespace        string
+	Labels           map[string]string
+	MHzPerCore       float64
+	Cores            int
+	CpuLimitHz       float64
```
Comment (justenwalker) [inline metrics/metrics.go]: style: Change to all-caps: `CPULimitHz`

Comment (ddreier) [reply metrics/metrics.go]: Done!

**Comment by justenwalker** [comment]
Merged! Thanks again @ddreier!

**Comment by ddreier** [comment]
Happy to help!

**Pull request merged**